watchers/chaos_pod_handler.go

```diff
@@ -42,7 +42,7 @@ func NewChaosPodHandler(disruption *chaosv1beta1.Disruption, recorder record.Eve
 }
 
 // OnAdd is a handler function for the add of the chaos pod
-func (w ChaosPodHandler) OnAdd(_ interface{}) {
+func (w ChaosPodHandler) OnAdd(_ interface{}, _ bool) {
```

Comment:
what does this bool do?

Reply:
haha, well I'm not sure. It's `isInitialList`, in the Handler interface, but there's almost no explanation of it in `client-go/tools/cache/controller.go`: `- OnAdd is called when an object is added.` 

elsewhere I can see this parameter explained briefly:
> AddFunc accepts the isInInitialList parameter, for propagating HasSynced.

So I check HasSynced
> Also used to tell if the handler is synced (has had all items in the initial list delivered).

and so I decided that worst case scenario, we're doing duplicate work? Truly though, I wasn't sure if these watchers would keep working until I ran a few big disruptions with this and it went fine